static/js/common/upload-addon.js

```diff
@@ -382,7 +382,7 @@
                                 $upload_field.trigger("upload_finished", [file]);
                             }
                         });
-                    }, 1000);
+                    }, 5000);
```

Comment:
> reduce the chance we're reading while the server is writing

That shouldn't be a problem. MySQL will block the read until the transaction is committed, which is what we want. We just need to optimize the speed of committing the transaction.